docs/metrics/customizing-the-sdk/README.md

```diff
@@ -335,6 +335,61 @@ by using Views.
 
 See [Program.cs](./Program.cs) for a complete example.
 
+### Explicit Bucket Histogram Aggregation with .NET 9 Advice API
```

Comment:
I doubt if this doc the right place, this doc is focusing on how to customize the SDK. Advice API is part of the instrumentation.